server/views/pages/offenceCodeProtectedCharacteristics.njk

```diff
@@ -0,0 +1,106 @@
+{% extends "../partials/layout.njk" %}
+{% from "../partials/breadCrumb.njk" import breadcrumb %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set title = "Offence details" %}
+
+{% block beforeContent %}
+  {{ breadcrumb() }}
+{% endblock %}
+
+{% block content %}
+  {% if errors | length %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors,
+      attributes: { "data-qa": "error-summary" }
+      }) }}
+  {% endif %}
+
+{% set items =
+[
+    {
+      value: ProtectedCharacteristicsTypes.AGE,
+      text: "Age"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.DISABILITY,
+      text: "Disability"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.GENDER_REASSIGN,
+      text: "Gender reassignment"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.MARRIAGE_AND_CP,
+      text: "Marriage and civil partnership"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.PREGNANCY_AND_MAT,
+      text: "Pregnancy and maternity"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.RACE,
+      text: "Race"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.RELIGION,
+      text: "Reglion or belief"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.SEX,
+      text: "Sex"
+    },
+    {
+      value: ProtectedCharacteristicsTypes.SEX_ORIENTATION,
+      text: "Sexual orientation"
+    }
+  ] %}
```

Comment:
Could set these in the controller and iterate through in the njk if that's preferable

Reply:
Might be able to jsut send through the enum and then use a njk filter to set the text based off the enum